packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -85,7 +85,6 @@ export default class QuanticGeneratedAnswer extends LightningElement {
    */
   @api withToggle = false;
   /**
-   * @internal
```

Comment:
In Quantic we use the tag `@api` in the JS doc for public properties,
it should be something like this:
```
  /**
   * @api
   * The unique identifier of the answer configuration to use to generate the answer.
   * @type {string}
   */
  @api answerConfigurationId;
```

Reply:
Thx @mmitiche